test/fixtures/extend/circular.json

```diff
@@ -134,7 +134,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "[\n  [],\n  []\n]"
+                              "content": "[\n  null,\n  null\n]"
```

Comment:
I cannot see how this example can be correct. There isn't a `nullable` type attributes on any structures in the source document so cannot be any `null` values present.